crates/holochain/src/core/state/metadata/status.rs

```diff
@@ -0,0 +1,34 @@
+use std::collections::HashSet;
+
+use holochain_zome_types::validate::ValidationStatus;
+
+#[derive(Debug, Clone, derive_more::Constructor, derive_more::From)]
+pub struct DisputedStatus {
+    status: HashSet<ValidationStatus>,
+}
+
+impl DisputedStatus {
+    /// Resolve a potentially disputed status by
+    /// considering everything except a set with only valid as invalid.
```

Comment:
@freesig why wouldn't the agent receiving diverging responses from authorities run validation itself and warrant everyone it disagrees with?

Reply:
They will once we have warrants. But there's still sometime where they can be in dispute while your running that validation and sending warrants etc.